tfhe/src/shortint/engine/server_side/shift.rs

```diff
@@ -70,7 +70,8 @@ impl ShortintEngine {
             self.unchecked_scalar_left_shift_assign(ct, shift)?;
         } else {
             let modulus = server_key.message_modulus.0 as u64;
-            let acc = self.generate_lookup_table(server_key, |x| (x << shift) % modulus)?;
+            let acc =
+                self.generate_msg_lookup_table(server_key, |x| x << shift, ct.message_modulus)?;
```

Comment:
this changes the modulus used here and mismatches the one in the call to after_left_shift below

Reply:
Do we support applying LUT with `sk.message_modulus != ct.message_modulus`?